library/dokku_network_property.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_network_property
+short_description: Set or clear a network property for a given dokku application
+options:
+  global:
+    description:
+      - Whether to change the global network property
+    default: False
+    aliases: []
+  app:
+    description:
+      - The name of the app. This is required only if global is set to False.
+    required: True
+    default: null
+    aliases: []
+  property:
+    description:
+      - |
+        The network property to be be modified. This can be any property network
+        property supported by dokku (e.g., `initial-network`, `attach-post-create`,
+        `attach-post-deploy`, `bind-all-interfaces`, `static-web-listener`, `tld`).
+    required: True
+    default: null
+    aliases: []
+  value:
+    description:
+      - The value of the network property (leave empty to unset)
+    required: False
+    default: null
+    aliases: []
+author: Philipp Sessler
+requirements: [ ]
+"""
+
+EXAMPLES = """
+- name: Associates a network after a container is created but before it is started
+  dokku_network_property:
+    app: hello-world
+    property: attach-post-create
+    value: example-network
+
+- name: Associates the network at container creation
+  dokku_network_property:
+    app: hello-world
+    property: initial-network
+    value: example-network
+
+- name: Setting a global network property
+  dokku_network_property:
+    global: true
+    property: attach-post-create
+    value: example-network
+
+- name: Clearing a network property
+  dokku_network_property:
+    app: hello-world
+    property: attach-post-create
+"""
+
+
+def dokku_network_property_set(data):
+    is_error = True
+    has_changed = False
+    meta = {"present": False}
+
+    command = "dokku network:set {0} {1} {2}".format(
```

Comment:
I suggest you check whether both `app` and `global` attributes are set and return an error if they are.

My feeling is that constructing these command strings with if blocks step by step is easier to understand and maintain, but feel free to keep this more compact notation.